source/includes/_resource.collections.md

```diff
@@ -229,6 +234,25 @@ vhx.collections.create({
       must must be a url of an image, hosted on vhx, prefixed with the text <code>image_url:</code>. 
       </td>
     </tr>
+    <tr class="is-internal text-2 border-bottom border--light-gray">
+      <td>
+        <strong class="is-block text--navy">plans</strong>
+        <span class="text--transparent text-3">array of strings</span>
+      </td>
+      <td>
+      An array of plan types that you can use to set collection availability. 
+      Values can be one or more of the following strings: <code>public</code>, <code>free</code>,
+      <code>standard</code>.<br><br>
+
+      The <code>public</code> plan makes the collection object available for free, without registration 
```

Comment:
I know we discussed IRL, but could use another IRL discussion to talk out something when available!